### ISSUE TITLE
[BUG] This repo is never checked

### ISSUE THREAD

**Issue by YodaLightsabr** [npm/npm-expansions#3697]
# What / Why

> This repo has not been modified since July of last year (#3488), despite the hundreds of pull requests.

## When

* Whenever I click on the [Pull requests](https://github.com/npm/npm-expansions/pulls), I see hundreds of pull requests that are open and neglected.
* Every day many new pull requests are created and ignored

## Where

* The pull requests tab

### Current Behavior

* Pull requests are neglected

### Steps to Reproduce

* Make a pull request
* Wait a few months

### Expected Behavior

* Make a pull request
* Wait a few days

## Who

* The NPM team

## References

* Related to every pull request that is open

**Comment by Explosion-Scratch** [comment]
bug lol

**Comment by YodaLightsabr** [comment]
Tbh, it bugs me

**Comment by ghost** [comment]
Npm busy with tryina fight yarn xD

**Comment by YodaLightsabr** [comment]
I'm very tempted to reference every open pull request under the references section. ![All of the open pull requests referenced in an image](https://yodacode.xyz/Fde7167aa5b.png)

**Comment by Explosion-Scratch** [comment]
![image](https://user-images.githubusercontent.com/61319150/153421492-7b1c2a79-74e6-40bf-8141-c92d28b7012d.png)

**Comment by TheDechev** [comment]
:fire: March 2022 hot update :fire:
- Number of pull requests :arrow_up:
- Number of merges :zero:

**Comment by Explosion-Scratch** [comment]
> Npm busy with tryina fight yarn xD

and pnpm

**Comment by ghost** [comment]
xD

**Comment by a-a-GiTHuB-a-a** [comment]
Eyy, this is a good one! This will definitely be added no way sirree will be this be ignored
(waits for the rest of their life for nothing)

**Comment by ghost** [comment]
> Eyy, a good one! This will definitely be added no way sirree will be this be ignored (waits like a year)

xD I quit using nodejs and started using java... and bro I have to tell you.. the performance was amazing!!!! don't listen to those people who lie about java being "slow" it ran so fast that I crashed my PC with a loop. because the threads run very fast and yea... it means its not limited like java xD

**Comment by a-a-GiTHuB-a-a** [comment]
You just said "java is not limited like java"
WHAT!?

**Comment by ghost** [comment]
oops java is not limited like javascript

**Comment by ghost** [comment]
does anyone have discord xD this issue has gotten pretty off topic I actually don't even think NPM willever take this issue into consideration

**Comment by YodaLightsabr** [comment]
I don't expect them to, I just thought this would be funny for people to stumble across.
Also, quick poll (👍 / 👎): should I add every open pull request to the references section?

**Comment by ghost** [comment]
im just gona follow everyone involved in this post and hope to make some friends xD

**Comment by akcumeh** [comment]
> Eyy, a good one! This will definitely be added no way sirree will be this be ignored (waits like a year)

I added a PR with 47 new NPM ideas and was feeling good about that till I saw the number of open PRs. D:

**Comment by a-a-GiTHuB-a-a** [comment]
> > Eyy, a good one! This will definitely be added no way sirree will be this be ignored (waits like a year)
> 
> xD I quit using nodejs and started using java... and bro I have to tell you.. the performance was amazing!!!! don't listen to those people who lie about java being "slow" it ran so fast that I crashed my PC with a loop. because the threads run very fast and yea... it means its not limited like java xD

Java is compiled into a native executable. Javascript is interpreted with one. That means JS is inherently slower. It's more feature-dense, though, and no compilation step is needed.

**Comment by ghost** [comment]
tf, java is NOT compiled to native. Also im now a C++ dev XDDDD super painful to use but Im good with it. also Java is rediculously slower compared to nodejs since it does not have JIT

**Comment by ghost** [comment]
yo any of u guys got Discord? at this point NPM clearly doesnt care about US THE POEPLE lmao

**Comment by ghost** [comment]
They probably even forgot existence of this repo or something, anyways if they don't care about the repo it would be better idea to archive it I believe, there is already a ton of fun messages.

**Comment by amol234545** [comment]
they should make a bot to do this so they can tell yarn is not better than npm

**Comment by a-a-GiTHuB-a-a** [comment]
Hundreds? Try thousands.

**Comment by amol234545** [comment]
> > Npm busy with tryina fight yarn xD
> 
> and pnpm
It literally has npm in it's name

**Comment by GoldenretriverYT** [comment]
> since it does not have JIT

V8 does do partial JIT compilation too...

**Comment by a-a-GiTHuB-a-a** [comment]
What is JIT, and why is it important?

**Comment by santi100a** [comment]
_Replying to @a-a-GiTHuB-a-a:_

> What is JIT, and why is it important?

JIT stands for **J**ust-**I**n-**T**ime Compilation. It is basically an approach to code execution used by, for instance, the V8 engine upon which Node.js and Chromium rely to execute JavaScript. Here, code is translated to executable format, then run in parallel as it's being read. This gives us the advantages of compiled-like speed and interpreted-like developer experience.

**Comment by X-Engineer-001** [comment]
Wait...... I should've realize this when I noticed the last edit time on expansions.txt......
So dump of me to send so many pull requests just thinking it's fun......

**Comment by a-a-GiTHuB-a-a** [comment]
The big problem with merging the 99999999 PRs is that each time, the NPM devs have to sort the answers (since the new expansions may conflict in position), which gets tedious.

**Comment by up201306506** [comment]
@a-a-GiTHuB-a-a don't be ridiculous. They could easily just have a pre-commit hook sort the list for them. The only and real reason is that this is extremely low priority and the key holder already has enough.

Honestly, with these fluff pet repos, you make it a small event if you want it getting attention at all. I'd have made it a team exercise to have everyone who works for npm, Inc take five minutes and choose a PR to merge on the day of the library's anniversary. That gives everyone some coffee talk too.
